nostr.ts

```diff
@@ -376,3 +302,4 @@ export * from "./websocket.ts";
 export * from "./event.ts";
 export * from "./key.ts";
 export * from "./relay.interface.ts";
+export * as v2 from "./v2.ts";
```

Comment:
After releasing the new version, when depending on updating this version, we need to modify the imported v2.

Reply:
yes